cmd/notary/integration_test.go

```diff
@@ -856,11 +858,11 @@ func TestClientDelegationsPublishing(t *testing.T) {
 	require.NotContains(t, output, "No delegations present in this repository.")
 
 	// remove the targets key to demonstrate that delegates don't need this key
-	keyDir := filepath.Join(tempDir, "private", "tuf_keys")
-	require.NoError(t, os.Remove(filepath.Join(keyDir, "gun", targetKeyID+".key")))
+	keyDir := filepath.Join(tempDir, "private")
+	require.NoError(t, os.Remove(filepath.Join(keyDir, targetKeyID+".key")))
 
 	// Note that we need to use the canonical key ID, followed by the base of the role here
```

Comment:
we should update this comment to explain that delegation keys need not have any PEM headers to specify the role, given that the filename is the canonical key ID